nexus/tests/integration_tests/updates.rs

```diff
@@ -482,29 +501,62 @@ fn assert_error_message_contains(
     Ok(())
 }
 
-// =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
-
-// Tests that ".." paths are disallowed by dropshot.
-#[tokio::test]
-async fn test_download_with_dots_fails() {
-    let cptestctx =
-        test_setup::<omicron_nexus::Server>("test_download_with_dots_fails", 0)
-            .await;
-    let client = &cptestctx.internal_client;
-
-    let filename = "hey/can/you/look/../../../../up/the/directory/tree";
-    let artifact_get_url = format!("/artifacts/{}", filename);
```

Comment:
I removed this test because we already test for directory traversal rejection in the console API integration tests:

https://github.com/oxidecomputer/omicron/blob/cbae98c3d48f8ecd77fc750d15daf1bdc6a72a2a/nexus/tests/integration_tests/console_api.rs#L375-L385

The `/artifacts/{..path}` internal API is long gone.